README.md

```diff
@@ -15,32 +14,36 @@ First you need to have [Ruby](https://www.ruby-lang.org/) (at least v2.3) instal
 To run the app locally, clone this repository and `cd` into its directory:
 
 1. First clone this repository and `cd` into its directory:
-   ```
-   git clone https://github.com/TwilioDevEd/block-spam-calls-ruby.git
-   cd block-spam-calls-ruby
+
+   ```bash
+   $ git clone https://github.com/TwilioDevEd/block-spam-calls-ruby.git
```

Comment:
I find it more useful not having the '$' at beginning of a shell example line unless you include the output as well. It's easier to copy without it and with the context and Github formatting people can understand it's a shell instruction. What do you think?

Reply:
It's a good observation @hortega. From the UX perspective, it will be easier to copy the instructions without `$`.